### ISSUE TITLE
BUG REPORT [5.0pre3] - Assigning articulation to multiple selected tracks (with Reaticulate enabled) results in error message

### ISSUE THREAD

**Issue by seventh-sam** [jtackaberry/reaticulate#156]
My other bug report for 5.0pre3:

EDIT: Running REAPER v6.45, Windows 10 (Up to date)

Assigning an articulation to multiple Reaticulate-enabled tracks (a new feature, iirc) does not seem to work.  See the screenshot below.

The following applies:
- All tracks have Reaticulate.jsfx enabled with the same bank (Tokyo Scoring Strings) loaded.
- All tracks have a Kontakt VSTi in the FX chain after the jsfx script
- The bug occurred regardless of which track was selected and whether or not multiple-track MIDI editing was enabled in REAPER.
- Reaticulate terminates after the error, but is able to be restarted immediately.

I've included the details below.  Let me know if there's anything else you need!

Text of error message:

CONSOLE:
```
01:11:47.252 [ERROR]   fatal: ...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:7637: attempt to index a nil value (local 'bank')
stack traceback:
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:186: in metamethod '__index'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:7637: in method '_insert_articulation'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:7702: in method 'activate_articulation'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:8313: in field 'onartclick'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:8348: in function <...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:8348>
	(...tail calls...)
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:2332: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:2702: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in method '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3058: in field '_handle_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3740: in method '_handle_window_event'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3660: in method '_update'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3239: in function <...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:3239>
	[C]: in function 'xpcall'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:191: in field 'call'
	...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:199: in function <...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:199>
```
REASCRIPT ERROR WINDOW:
`...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:187: ...pData\Roaming\REAPER\Scripts\Reaticulate\reaticulate.lua:7637: attempt to index a nil value (local 'bank')`

Image:

![image](https://user-images.githubusercontent.com/70399072/149888818-b937aa33-d791-432e-8d4c-f39fa4b70f35.png)

**Comment by jtackaberry** [comment]
Thanks @seventh-sam.  This is a silly regression I introduced with some recent (apparently poorly tested) refactoring.  Easy fix. :)

**Comment by jtackaberry** [comment]
I just released 0.5.0-pre4 which fixes this.
